### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by OLIPLICHE**
**I create a form of UI for my school library app as follow:**

1. I Create the main file that serves as a console app entry-point.
2.  In the main, define a method that should: 
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    print 'Specialization: '
+    specialization = gets.chomp
+
+    teacher = Teacher.new(specialization, age, name)
+    @people << teacher
+
+    puts 'Teacher created successfully'
+    sleep 0.75
+    menu
+  end
```
Comment (HillaryOkello) [inline main.rb:126]: - [x] Your `create_a_student` and `create_a_teacher` methods are not working as they should. I am getting the following error an `argument error` when I try to add a student or a teacher. 
student
 Please check and make sure all the arguments are well set.

**Review by HillaryOkello — CHANGES_REQUESTED** [review]
# Hello @OLIPLICHE, 

## To Highlight
- Clean code ✅ 
- Descriptive pull request ✅ 
#
Good job so far! You've passed almost all the checklists for this project.
There is just one issue that you still need to work on to go to the next project but you are almost there! You can do it 💪🏻 


## Required Changes ♻️
- [x] Always make sure to use descriptive commit messages instead of `update main` you could write the specific update you made. The commit message should be written in the imperative tense and capitalized
commits


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (OLIPLICHE) [reply main.rb:126]: Sorry, I did not check the code. I was dosing off while working. I forgot to run the test

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Paka, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**